build.xml

```diff
@@ -195,10 +212,6 @@
   
   <target name="build.modules" depends="init, clean, compile.module.pearplanner" description="build all modules"/>
   
-  <target name="all" depends="build.modules" description="build all"/>
```

Comment:
The "all" target should be left in place and remain the default action.  Most developers will expect the default target to only build (i.e., compile and generate artifacts) and possibly run tests.  Launching the application is typically not a default action and I don't think it is appropriate for our project.